src/common/websocket/config.js

```diff
@@ -72,6 +80,10 @@ const getAppId = () => {
             app_id = user_app_id
         } else if (isLocalHost()) {
             app_id = domain_config.local.app_id
+        } else if (url) {
+            if (isBrowser() && cell_xpert.staging.hostname === window.location.hostname) {
```

Comment:
Since we will refactor this later, let's clean it now. Let's change the condition to check if the current page is `signup-affiliates` then we apply the correct app id